frontend/src/routes/Infrastructure/Clusters/ClusterSets/ClusterSetDetails/ClusterSetDetails.test.tsx

```diff
@@ -1065,6 +1071,7 @@ describe('ClusterSetDetails page', () => {
     await typeByTestId('awsAccessKeyID', mockManagedClusterNoCredentialsSecret.data!.aws_access_key_id)
     await typeByTestId('awsSecretAccessKeyID', mockManagedClusterNoCredentialsSecret.data!.aws_secret_access_key)
     await typeByTestId('global-net-cidr', mockManagedClusterNoCredentialsSubmarinerConfig.spec?.globalCIDR!)
+    await clickByLabel('Use Custom Submariner Subscription')
```

Comment:
Check here for cases as well. Make sure it is consistent.

Reply:
Changed to lower case expect for the starting word.